mise.toml

```diff
@@ -0,0 +1,62 @@
+#:schema https://mise.jdx.dev/schema/mise.json
+
+[tools]
+uv = "0.5.1"
+
+[tasks."setup"]
+description = "Set up the dev environment (install Rust, etc)"
+run = [
+  "rustup toolchain install 1.84",
```

Comment:
(Non-blocking) This might be controversial, but I'd just use stable. Rust is getting pretty good at backward-compatibility, e.g. clippy is aware of which version of Rust is the MSRV and doesn't suggest inappropriate hints. OTOH, refactoring tools and clippy itself get better in newer versions, as does compilation speed, etc.